deep-linking/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "bpmn-js-example-commonjs",
+  "version": "0.0.0",
+  "description": "An example how to bundle an app using bpmn-js for the browser",
+  "main": "src/app.js",
+  "scripts": {
+    "all": "run-s bundle open",
+    "bundle": "webpack",
+    "bundle:watch": "webpack -w",
+    "open": "opn ./public/index.html",
+    "dev": "npm run bundle:watch"
```

Comment:
My assumption was that `npm start` spawns a demo instance (or `npm run dev` would).

But apparently this is not a pattern for our examples?

Reply:
It does seem like there is no pattern, I just copied and adjusted another example. I'm fine with introducing it though